### DISCUSSION TITLE
nvim version and "buftype" error

### DISCUSSION THREAD

**Discussion by lcycon** [vscode-neovim/vscode-neovim#2122] [General]
Figured I'd drop this for anyone else struggling. If you are getting a mysterious `error: E382: Cannot write, 'buftype' option is set` error without any nvim extensions/remappings/etc, it seems making sure you have `nvim 0.10.0` solves all!

**Comment by justinmk** [comment]
This is another case in favor of [increasing our minimum required Nvim](https://github.com/vscode-neovim/vscode-neovim/pull/2092#discussion_r1632437219) (and showing a "update Nvim" warning) :)
